docs/_templates/layout.html

```diff
@@ -6,7 +6,8 @@
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>{{ title|striptags|e }}{{ titlesuffix }}</title>
-  {%- block extrahead %} {% endblock %}
+  <!-- begin extra head -->
+
   <!-- end extra head -->
```

Comment:
```suggestion
  {%- block extrahead %} {% endblock %}
```
This is needed for search to work